plugins/3scale-backend/dist-dynamic/README.md

```diff
@@ -0,0 +1,74 @@
+# 3scale Backstage provider
```

Comment:
@davidfestal dist-dynamic dir is something that would be generated or in the future plugin authors/maintainers need to add this manually for all backend plugins?

Reply:
You're right. It's something I have to look into today. I should have opened the PR as WIP.

Reply:
In fact only the `package.json` and `yarn.lock` in `dist-dynamic` should be committed back, after being generated by a local build.

Reply:
I updated the .gitignore accordingly